index.d.ts

```diff
@@ -590,10 +583,7 @@ declare module '@primer/components/lib/CircleOcticon' {
   import {CircleOcticon} from '@primer/components'
   export default CircleOcticon
 }
-declare module '@primer/components/lib/StyledOcticon' {
```

Comment:
It doesn't look like `@primer/styled-octicons` exports types so I think we're going to need to do that over in that repo before merging this PR

Reply:
Just kidding, looks like it does: https://github.com/primer/octicons/blob/master/lib/octicons_styled/script/build.js#L30